.tool-versions

```diff
@@ -1 +1 @@
-ruby 3.1.0
+ruby 3.4.2
```

Comment:
Hi Team, I was setting up the repo. How about updating `.ruby-version` to `3.4.2`?

There's also a reference to Rails 7 within the [Readme line 36](https://github.com/hellobrightline/interview_project/blob/main/README.md?plain=1#L36).

Reply:
Thanks @tlynam !

Reply:
Sure thing @gvarela!